docs/en/interfaces/jdbc.md

```diff
@@ -7,7 +7,3 @@ sidebar_label: JDBC Driver
 # JDBC Driver
 
 Use the [official JDBC driver](https://github.com/ClickHouse/clickhouse-jdbc) (and Java client) to access ClickHouse from your Java applications.
```

Comment:
If it's the only recommendation, I'm wondering if we should remove the page and set a URL redirect to https://clickhouse.com/docs/integrations/java/jdbc-v2

Reply:
you'll have to teach me that trick 😅

Reply:
> you'll have to teach me that trick 😅

I *think* it should be done here https://github.com/ClickHouse/clickhouse-docs/blob/main/vercel.json#L456-L458